api/src/main/java/io/opentelemetry/context/NoopScope.java

```diff
@@ -18,18 +18,14 @@
 
 /** A {@link Scope} that does nothing when it is created or closed. */
 public final class NoopScope implements Scope {
-  private static final Scope INSTANCE = new NoopScope();
-
-  private NoopScope() {}
-
   /**
-   * Returns a {@code NoopScope}.
+   * A singleton instance of {@code NoopScope}.
    *
-   * @return a {@code NoopScope}.
+   * @since 0.1.0
    */
-  public static Scope getInstance() {
-    return INSTANCE;
-  }
+  public static final Scope INSTANCE = new NoopScope();
```

Comment:
Defining this as `INSTANCE` for now. Can be changed as part of #309 

All `Tagger`, `Meter`, `Tracer` uses `getInstance`.